pkg/client/response.go

```diff
@@ -44,3 +44,8 @@ func (r *Response) Bytes() []byte {
 func (r *Response) String() string {
 	return string(r.body)
 }
+
+// Header returns the header value.
+func (r *Response) Header(name string) string {
+	return r.header.Get(name)
```

Comment:
Are you sure header is not nil at this point?

Reply:
Added a check to be safe.